### PR TITLE
Update various constaints & dependencies

### PR DESCRIPTION
**Pull request by Chris00**
- Cairo2
- FreeTDS
- Gammu
- Integration1D
- opamfu

### CODE REVIEW

File: packages/freetds/freetds.0.4/opam [hunk 1]
```diff
@@ -20,3 +26,10 @@ build: [
 remove: [["ocamlfind" "remove" "freetds"]]
 depends: ["ocamlfind"]
 install: [make "install"]
+depexts: [
+  [["centos"] ["autoconf" "automake" "freetds-devel"]]
+  [["debian"] ["autoconf" "automake" "freetds-dev"]]
+  [["fedora"] ["autoconf" "automake" "freetds-devel"]]
+  [["ubuntu"] ["autoconf" "automake" "freetds-dev"]]
+  [["osx" "homebrew"] ["autoconf" "automake" "freetds"]]
```
Comment (kit-ty-kate) [inline packages/freetds/freetds.0.4/opam]: you might maybe want to depend on the `conf-autoconf` package instead. It would avoid redundancies.

**Comment by kit-ty-kate** [comment]
Sorry for the delay but there are a lot of failures in `V1.2 Compilers` that I don't understand. Maybe you can maybe sens of them ? If you can't I'll go ahead and merge this.

**Comment by hcarty** [comment]
This seems ok to merge.

**Comment by kit-ty-kate** [comment]
Mmh ok let's merge this and will worry later about the failures.
Sorry for the delay :/

**Comment by Chris00** [comment]
@kit-ty-kate Sorry but did not find the time to investigate.  Thanks for merging.

**Pull request merged**